include/rocksdb/options.h

```diff
@@ -57,6 +57,21 @@ class FileSystem;
 struct Options;
 struct DbPath;
 
+// To indicate the file types
```

Comment:
Perhaps

    // The types of files RocksDB uses in a DB directory. (Available for advanced options.)

Reply:
Perhaps the enum should be defined in "advanced_options.h" instead of options.h?

I am not in favor of classes that are defined in the public API only having forward declarations.

Reply:
If we want to expose file type as public, how about considering the file "include/rocksdb/types.h"? According to the file's comment, it is for all public custom types.

Reply:
Sure, I will move to "include/rocksdb/types.h", it makes more sense for me.